src/Provider.ts

```diff
@@ -646,6 +647,16 @@ export const Provider: React.FC = ({ children }) => {
     }),
     []
   )
+  useDebugValue(
```

Comment:
As far as I know, `useDebugValue` sets a label for custom hooks, so it must be located whithin a hook. However, this is not under any hook but a component.

Reply:
I see. I wasn't aware that. Thanks for confirming. So, should we create a custom hook for this purpose? Sounds a bit weird, but let's try and see how it goes.